### PR TITLE
add -y flag to skip confirm

### PR DESCRIPTION
**Pull request by overcookedpanda**
Bash doesn't allow you to use long options, so I implemented `-y` which will allow a user to skip the confirmation for install and error if they add any other option.

#16

### CODE REVIEW

**Comment by johnnychen94** [comment]
Looks good. And there's a `read -p` in https://github.com/abelsiqueira/jill/blob/1c21109eb98cc26883a9fe9e956a90877d97f828/jill.sh#L45 that needs to be skipped as well.

Also, you might need to update the README.md before merging.

**Comment by overcookedpanda** [comment]
> Looks good. And there's a `read -p` in
> 
> https://github.com/abelsiqueira/jill/blob/1c21109eb98cc26883a9fe9e956a90877d97f828/jill.sh#L45
> 
> that needs to be skipped as well.

Thanks, I didn't catch that originally, but it has been updated now.

File: jill.sh [hunk 1]
```diff
@@ -41,13 +54,19 @@ function badfolder() {
...
+    if [[ ! $REPLY =~ ^[Yy] ]]; then
+      echo "Aborted"
+      exit 1
+    else
+      echo 'export PATH="'"$JULIA_INSTALL"':$PATH"' | tee -a ~/.bashrc
+      echo ""
+      echo "run 'source ~/.bashrc' or restart your bash to reload the PATH"
+      echo ""
+    fi
+  elif [[ "$SKIP_CONFIRM" == "1" ]]; then
```
[the remark this answers is not in the document]
Comment (overcookedpanda) [reply jill.sh]: I believe this is resolved in the last commit.

**Comment by overcookedpanda** [comment]
I'm not exactly sure why the travis test is failing, as it appears to fail on different checks each round.  The changes requested should be sufficient for it to pass all tests.

File: jill.sh [hunk 2]
```diff
@@ -41,18 +54,18 @@ function badfolder() {
...
+    echo
+    if [[ ! $REPLY =~ ^[Yy] ]]; then
+      echo "Aborted"
+      exit 1
+    fi
+  fi
     echo 'export PATH="'"$JULIA_INSTALL"':$PATH"' | tee -a ~/.bashrc
     echo ""
     echo "run 'source ~/.bashrc' or restart your bash to reload the PATH"
     echo ""
```
Comment (abelsiqueira) [inline jill.sh]: These lines need to be indented with 2 spaces now.

**Comment by abelsiqueira** [comment]
Thanks again. Just a small adjustment and we're done. The tests really are acting strange. I opened an issue (#19) about it.

**Comment by overcookedpanda** [comment]
I think we’re done now, glad I could help.

**Comment by abelsiqueira** [comment]
Thank you!

**Pull request merged**